stdlib/Printf/src/Printf.jl

```diff
@@ -626,11 +626,22 @@ end
 function fix_dec end
 function ini_dec end
 
+"maximal exponent for `Float64`s: `ceil(Int, log10(prevfloat(typemax(Float64))))`"
+max_integer_part_width() = 309
```

Comment:
Why are these functions?

Reply:
> Why are these functions?

It allows to reuse magic numbers in multiple places in this file. Should they be `const` globals instead ?

> Also, seems like the string should just be a comment?

Yes, we could make them a comment, but it's imo better to leave them as a docstring.

Reply:
> Should they be uppercased const globals instead ?

Yes, it seems like just consts is more lightweight than having to define a function for every single number.

> Any argument against?

The docstring has to be stored somewhere etc. This just seems like a completely standard scenario where you would use a comment.

Reply:
Allright, suggestions applied.